lua/image/utils/document.lua

```diff
@@ -10,6 +10,13 @@ local resolve_absolute_path = function(document_file_path, image_path)
   return absolute_image_path
 end
 
+local resolve_base64_image = function(document_file_path, image_path)
+  local tmp_b64_path = vim.fn.tempname()
+  vim.fn.writefile({ image_path }, tmp_b64_path)
+  os.execute("convert inline:" .. tmp_b64_path .. " " .. tmp_b64_path)
```

Comment:
Works great, but this os.execute opens users to command injection, `vim.uv.spawn` with an arg table would be enough to fix it.
The only other issue is that we'd always need `convert` to be available, could we do the conversion directly in Lua?

Reply:
Hi, I've just pushed a commit that uses native `vim` function `vim.base64.decode` to decode the image, save the image and return its path.

As a side note, `vim.fn.writeline` didn't seem to work very well with the decoded string. So, I've ended up migrating to `io`.